zip-xxxx.rst

```diff
@@ -0,0 +1,290 @@
+| **Common Transaction Fields**                                                                                                                                         |
++-----------------------------+--------------------------+----------------------------------------+---------------------------------------------------------------------+
+|``4``                        |``header``                |``uint32``                              |Contains:                                                            |
+|                             |                          |                                        |  * ``fOverwintered`` flag (bit 31, always set)                      |
+|                             |                          |                                        |  * ``version`` (bits 30 .. 0) – transaction version.                |
++-----------------------------+--------------------------+----------------------------------------+---------------------------------------------------------------------+
+|``4``                        |``nVersionGroupId``       |``uint32``                              |Version group ID (nonzero).                                          |
++-----------------------------+--------------------------+----------------------------------------+---------------------------------------------------------------------+
+|``4``                        |``nConsensusBranchId``    |``uint32``                              |Consensus branch ID (nonzero).                                       |
++-----------------------------+--------------------------+----------------------------------------+---------------------------------------------------------------------+
+|``4``                        |``lock_time``             |``uint32``                              |Unix-epoch UTC time or block height, encoded as in Bitcoin.          |
++-----------------------------+--------------------------+----------------------------------------+---------------------------------------------------------------------+
+|``4``                        |``nExpiryHeight``         |``uint32``                              |A block height in the range {1 .. 499999999} after which             |
+|                             |                          |                                        |the transaction will expire, or 0 to disable expiry.                 |
+|                             |                          |                                        |[ZIP-203]                                                            |
++-----------------------------+--------------------------+----------------------------------------+---------------------------------------------------------------------+
+|``8``                        |``fee``                   |``int64``                               |The fee to be paid by this transaction, in zatoshis.                 |
```

Comment:
```suggestion
|``8``                        |``fee``                   |``int64``                               |The non-negative fee to be paid by this transaction, in zatoshis.                 |
```

Reply:
Or are negative values allowed in coinbase transactions?

Reply:
Teor's comment refers to a brainwave I had during the ZIP sync today, which is that a negative fee in a coinbase transaction could represent the fees collected from other transactions in the block. This would be more regular, because:
* every transaction would balance exactly;
* the sum of `fee` fields in each block would be zero.